explore/src/main/webapp/index.html

```diff
@@ -14,6 +14,11 @@
       var global = window;
     }
   </script>
+  <script type="module">
+    import process from "process";
+
+    window.process = process;
```

Comment:
Where is `process` used?

Reply:
That's the secret sauce. react-markdown needs it but it is not provided in the dev enviroment in vite so it needs a polyfill